backend/lobby/consumers.py

```diff
@@ -253,14 +260,42 @@ def gamelog_send_handler(self, data):
 
         self.send_server_response_to_client(username, room, mud_gamelog_input)
 
-    # def roll_dice_handler(self, data):
-    #     pass
+    def roll_dice_handler(self, data):
+        pass
+
+    def card_store_request_handler(self, data):
+        username = data['user']
+        room = data['room']
+        # game = GameState.objects.get(room_name=room)
+        # state: BoardGame = pickle.loads(game.board)
+        selected_cards_ui_message = state.deck_handler.json_store()
+
+        self.send_rolls_to_client(username, room, selected_cards_ui_message)
 
     commands = {
         'init_user_request': init_chat_handler,
         'gamelog_send_request': gamelog_send_handler,
         'selected_dice_request': selected_dice_handler,
         # 'roll_dice_request': roll_dice_handler,
         'return_dice_state_request': return_dice_state_handler,
-        'end_turn_request': end_turn_handler
+        'end_turn_request': end_turn_handler,
+        'card_store_request': card_store_request_handler
     }
+
+
+def test_get_phony_card_market():
```

Comment:
These methods should be removed now that we have direct access to the correct card market

Reply:
Removed those methods